cluster/vmpreset-windows.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: kubevirt.io/v1alpha1
+kind: VirtualMachinePreset
+metadata:
+  name: windows-server-2012r2-preset
+  selector:
+    matchLabels:
+      os-short-id: win2k12r2
```

Comment:
+1

We need to start a user-guide - or here in this rpeo - design doc to specify these annotations and their semantics and realtionships to libosinfo.

Reply:
@fabiand  - it on the way, under the user guide repo.